worlds/dkc3/Names/LocationName.py

```diff
@@ -294,7 +294,7 @@
 blue_region        = "Blue's Beach Hut Region"
 blizzard_region    = "Bizzard's Basecamp Region"
 
-lake_orangatanga_region = "Lake_Orangatanga"
+lake_orangatanga_region = "Lake Orangatanga"
```

Comment:
since i don't see it mentioned in your PR description is this also a bug fix?

Reply:
ah it was surrounded in () that's why i didn't see it. then i suppose i just need to ask if it's used in code anywhere as a precaution